gsl_programmation/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from dsfr.forms import DsfrBaseForm
+
+
+class SimulationForm(DsfrBaseForm):
+    title = forms.CharField(
+        label="Titre de la simulation", max_length=100, required=True
+    )
+    dotation = forms.ChoiceField(
+        label="Dotation associée",
+        choices=[
+            ("", "Choisir un fond de dotation"),
```

Comment:
Ici "fonds" prend un S même au singulier. [Si, si.](https://www.cnrtl.fr/definition/fonds)


```suggestion
            ("", "Choisir un fonds de dotation"),
```